src/app/src/main/java/com/cmput301f21t09/budgetprojectname/models/HabitModel.java

```diff
@@ -202,10 +205,23 @@ private HabitModel(String id, String title, String reason, Date startDate, boole
                 .whereArrayContainsAny("schedule", new HabitScheduleModelFactory().getQueryForToday())
                 .get().addOnCompleteListener(task -> {
             if (task.isSuccessful()) {
+                List<DocumentSnapshot> docs = task.getResult().getDocuments();
                 List<HabitModel> models = new ArrayList<>();
                 HabitModelMapParser parser = new HabitModelMapParser();
-                for (QueryDocumentSnapshot document : task.getResult()) {
-                    models.add(parser.parseMap(document.getData(), document.getId()));
+                for (DocumentSnapshot doc : docs) {
+                    Log.d("YEET", "DocumentSnapshot data: " + doc.getData());
```

Comment:
Please change the tag to something more relevant

Reply:
oops I forgot about that. I'll delete it LOL